lib/tilex/notifications/notifiers/twitter.ex

```diff
@@ -16,7 +18,40 @@ defmodule Tilex.Notifications.Notifiers.Twitter do
     :ok
   end
 
-  def send_tweet(text) do
-    ExTwitter.update(text)
+  def send_tweet(message) do
+    url = "https://api.x.com/2/tweets"
+
+    params = %{
+      "text" => message
+    }
+
+    headers =
+      oauth_headers("post", url) ++
+        [{"Content-Type", "application/json"}]
```

Comment:
did it not work without the content type header?

Reply:
I just tried without it and it does work 👍

Reply:
awesome!